packages/firebase_analytics/firebase_analytics/lib/src/firebase_analytics.dart

```diff
@@ -125,14 +125,27 @@ class FirebaseAnalytics extends FirebasePluginPlatform {
     );
   }
 
-  /// Sets the applicable end user consent state. 'default' value for 'adStorageConsentGranted' & 'analyticsStorageConsentGranted' is 'true'
+  /// Sets the applicable end user consent state.
+  /// By default, no consent mode values are set.
+  ///
+  /// Default consents can be set according to the platform:
+  /// - [iOS][1]
+  /// - [Android][2]
+  ///
+  /// [1]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=ios#default-consent
+  /// [2]: https://developers.google.com/tag-platform/security/guides/app-consent?platform=android#default-consent
   Future<void> setConsent({
     bool? adStorageConsentGranted,
     bool? analyticsStorageConsentGranted,
+    bool? adPersonalizationSignalsConsentGranted,
+    bool? adUserDataConsentGranted,
```

Comment:
You still haven't allowed all the arguments to be passed through to web. I would also make it clear in the inline documentation (i.e above the `setConsent()` function here and in the platform interface) which parameters are available for each platform.

Reply:
Done. 7abec92